opentelemetry-sdk/CHANGELOG.md

```diff
@@ -8,6 +8,17 @@
 - **BREAKING**: [#2217](https://github.com/open-telemetry/opentelemetry-rust/pull/2217)
   - **Replaced**: Removed `{Delta,Cumulative}TemporalitySelector::new()` in favor of directly using `Temporality` enum to simplify the configuration of MetricExporterBuilder with different temporalities.
 - When creating new metric instruments, SDK would return a no-op instrument if the validation fails. [#2166](https://github.com/open-telemetry/opentelemetry-rust/pull/2166)
+- **Breaking change for Metrics users:** The `init` method used to create instruments has been renamed to `build`.
```

Comment:
Even though this is a part of API, I have updated the CHANGELOG in the SDK as well to make this change more noticeable for users. We could also post specifically about end user facing breaking changes on Slack and GitHub announcements when we release the new version of these crates.

Reply:
the next release really need a migration guide, consolidating all the breaking changes.